src/components/Message/Message.tsx

```diff
@@ -0,0 +1,20 @@
+import classes from 'classnames';
+import * as React from 'react';
+
+interface MessageProps {
+  /** The contents of the Message */
+  children?: React.ReactNode;
+  /** Three styles are available via this className pattern:
+   * `Message--{danger|info|success}`
+   */
+  className?: string;
```

Comment:
Should this type also include "null"?
```suggestion
  className?: string | null;
```

I've seen this pattern in other places in our code base where we have 
```
  className?: string | null | undefined;
```

Logan explained that we actually don't need the undefined because having the `?` after classname implicitly adds that to the type of className, but do we still need null?

Reply:
Good question! We do have some legacy code in pare_web that is include the `null` type -- but moving forward I think it's better to match how the native dom elements type their `className` prop as just `string | undefined`:

<img width="745" alt="Screen Shot 2022-02-18 at 13 15 47" src="https://user-images.githubusercontent.com/147870/154762262-4b8b46f5-6c2e-4662-9401-09df7e72f216.png">

So the proper way to type it is:

```ts
propName?: string;
```